packages/app/lib/ui/widgets/pagination_list.dart

```diff
@@ -201,6 +215,30 @@ class SliverPaginationBase<T> extends StatelessWidget {
   }
 }
 
+class FetchAll<T> extends StatelessWidget {
```

Comment:
New widget which fetches all pages of a collection query in one go, and passes the whole collection onto it's child via a custom builder method.